tealiumlibrary/src/main/java/com/tealium/core/consent/ConsentSharedPreferences.kt

```diff
@@ -3,6 +3,9 @@ import android.content.SharedPreferences
 import com.tealium.core.TealiumConfig
 import com.tealium.core.consent.ConsentManagerConstants.KEY_STATUS
 import com.tealium.core.consent.ConsentManagerConstants.KEY_CATEGORIES
+import com.tealium.core.consent.ConsentManagerConstants.KEY_LAST_SET
```

Comment:
Also consider rename to correspond to ConsentManager property. Either `KEY_LAST_UPDATED` or `KEY_LAST_CONSENT_UPDATE`